CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Change Log
 
+## Version 1.17.0
+* Release date: January 25, 2023
+* Release status: GA
+
+### What's new in 1.17.0
+* BREAKING CHANGE - Encryption is now Enabled by Default - https://github.com/microsoft/vscode-mssql/pull/17484
+	- Moving to Microsoft.Data.SqlClient 5.0.1 dependency with STS Update
+	- Connections should now be more sercure but some saved connections may be broken
```

Comment:
```suggestion
	- Connections should now be more secure but some saved connections may be broken
```

Reply:
The word "should" makes me itchy. "By default, saved connection profiles will connect with encryption and only accept trusted server certificates. This is a breaking change for some connections and some saved connection profiles may require updates to connect."

Reply:
Took Drew's updated phrasing